### ISSUE TITLE
verifyfiles function doesn't work

### ISSUE THREAD

**Issue by talgoldengoren** [speechbrain/speechbrain#1026]
When I try to verify 2 audio files with the function `verify_files()` with the next few lines of code:

```
from speechbrain.pretrained import SpeakerRecognition
verification = SpeakerRecognition.from_hparams(source="speechbrain/spkrec-ecapa-voxceleb", savedir="pretrained_models/spkrec-ecapa-voxceleb")
score, prediction = verification.verify_files("voxceleb1/wav/id10270/5r0dWxy17C8/00010.wav", "voxceleb1/wav/id10305/19OCRADWjgA/00006.wav")
print(score)
```

The resulting behavior is weird. 
It works like a decreasing counter.
Every time I run these lines:
```
score, prediction = verification.verify_files("voxceleb1/wav/id10270/5r0dWxy17C8/00010.wav", "voxceleb1/wav/id10305/19OCRADWjgA/00006.wav")
print(score)
```
The score is decreasing in `0.0001`.

So if I run a loop:
```
for i in range(10):
  score, prediction = verification.verify_files("voxceleb1/wav/id10270/5r0dWxy17C8/00010.wav", "voxceleb1/wav/id10305/19OCRADWjgA/00006.wav")
  print(score)
```

This is the result:
![image](https://user-images.githubusercontent.com/52013932/135765580-f09e2a7e-aaf1-4202-9f36-6884555c49ea.png)

I think It may be a problem in the `verify_files()` method, I tried it with many different files and from different computers and with different pretrained models ([ECAPA](https://huggingface.co/speechbrain/spkrec-ecapa-voxceleb), [XVectors](https://huggingface.co/speechbrain/spkrec-xvect-voxceleb)), it was always the same, a decreasing counter.

**Comment by Gastron** [comment]
I was able to reproduce this and I think I found the issue. The input normalization keeps updating. If we stop it from updating, by setting:
```python
verification.modules.mean_var_norm_emb.update_until_epoch = -1
```
The scores become constant.

@mravanelli should the input normalization be set to not update for these pretrained models? This can be achieved for instance with adding `update_until_epoch: -1` in [https://huggingface.co/speechbrain/spkrec-ecapa-voxceleb/blob/main/hyperparams.yaml#L34](https://huggingface.co/speechbrain/spkrec-ecapa-voxceleb/blob/main/hyperparams.yaml#L34)

**Comment by TParcollet** [comment]
@Gastron it should be set. Are you a member of the HF team ? If yes, you can edit directly from the website the yaml :)

**Comment by mravanelli** [comment]
Yes, we can freeze it!

**Comment by Gastron** [comment]
I've edited the embedding normalization on HuggingFace. Does this solve the issue for you as well, @talgoldengoren ?

**Comment by talgoldengoren** [comment]
Yes,  the score stays the same now.
Thank you  :) @Gastron
